classes/Layer/Layer.mjs

```diff
@@ -0,0 +1,41 @@
+export class Layer {
+    constructor(layer = {}) {
+        this.layer = layer
+        if(!this.layer.id) {
+            this.layer.id = Date.now() // ticket to replace with MongoDB ObjectID()
+        }
+    }
+    
+    get id() {
+        return this.layer.id
+    }
+
+    set id(id) {
+        this.layer.id = id
+    }
+
+    asJSON() {
+        return {
+            type: 'Range',
+            '@context': 'http://www.w3.org/ns/anno.jsonld',
```

Comment:
https://iiif.io/api/presentation/3.0/ Look at the Range here for a "real" example.

Reply:
Still incorrect